README.md

```diff
@@ -1 +1,139 @@
-# microcosm-fastapi
\ No newline at end of file
+# microcosm-fastapi
+
+A bridge between FastAPI and microcosm. Provide state-of-the-art speed of hosting microservices in Python along with dependency injection.
+
+## Top Features
+
+- Async compliant protocol to add new microservice functions
+- Specify API requests and responses with business-logic typehinting, while following strongly conventioned CRUD operations to access the database
+- Async postgres support using the latest SQLAlchemy 1.4 (still in beta), to support more concurrent client users with fewer CPU blocking requests
+- Automatic generation of interactive documentation, available on localhost:5000/docs when doing development work
```

Comment:
Note on this the current version of fastapi docs uses a external CDN for the js bundles which leaks info to 3rd party sites. See description here: https://github.com/tiangolo/fastapi/issues/2518

So some extra work is required before using these docs.

Reply:
@rh-glob Good callout. Put up a new commit with locally hosted docs.